### ISSUE TITLE
When will you fix the bug with the Error display effect on the “placebo”（or “9”） preset?

### ISSUE THREAD

**Issue by jiangzemin-xudamin** [ultravideo/uvg266#28]
I don't know why this problem has arisen.
Maybe it's not fixed yet? Maybe there's a problem with the version I downloaded?
uvg266 VVC v0.8.1-2d2301c
https://www.mediafire.com/file/rqs8q57j8l4mv6y

```powershell
md output
md temp
for %%a in (*.mkv *.mp4 *.webm *.mov *.avi *.rm *.rmvb *.flv *.3gp *.wmv) do (
md ".\temp\%%~na"
.\ffmpeg -hide_banner -i "%%~sa" -map 0:a:0 -vn -sn -c:a pcm_s32le -threads 32 -strict -2 -ac 2 -f wav - | .\exhale a ".\temp\%%~na\0.m4a"
.\ffmpeg -hide_banner -loglevel error -i "%%~sa" -map 0:v:0 -an -sn -pix_fmt yuv420p -strict -2 -f yuv4mpegpipe - | .\uvg266 -i - --input-file-format y4m --owf 58 --threads 32 --qp 35 --preset 9 -o ".\temp\%%~na\vvvv.266"
mp4box -add ".\temp\%%~na\vvvv.266" ".\temp\%%~na\mmmm.mp4"
mp4box -add ".\temp\%%~na\0.m4a" ".\temp\%%~na\mmmm.mp4"
.\ffmpeg -hide_banner -n -i ".\temp\%%~na\mmmm.mp4" -map 0:v -c:v copy -map 0:a -c:a copy ".\output\%%~na.mkv"
)
pause
rmdir /s/q temp
echo ————!!!Conversion completed!!!"————
pause
``` 

![PixPin_2024-12-05_18-32-02](https://github.com/user-attachments/assets/5467ede1-cb00-482c-aa11-dbba5150d497)
![PixPin_2024-12-05_18-32-41](https://github.com/user-attachments/assets/f090646c-239e-4fa0-9d32-797eab88ded9)

**Comment by jiangzemin-xudamin** [comment]
When the preset is "veryslow" (8), the display effect is normal.
![PixPin_2024-12-06_08-10-35](https://github.com/user-attachments/assets/ef138713-2eb1-4a55-9ff3-1b0954b90b36)

**Comment by Eric3495** [comment]
邮件发到~~

**Comment by jiangzemin-xudamin** [comment]
> IMHO it's not completely impossible that the encoding is correct but beyond the capabilities of the decoder?

@LigH-de Not really, I use MPV.NET, MPC-BE and PotPlayer and all have the same problem!

**Comment by jiangzemin-xudamin** [comment]
@Eric3495 你邮件发哪去啦？只有GitHub默认的邮件。

**Comment by LigH-de** [comment]
Today (with uvg266 0.8.1-391a564) the official presets syntax is:

```
      --preset <preset>      : Set options to a preset [medium]
                                   - ultrafast, superfast, veryfast, faster,
                                     fast, medium, slow, slower, veryslow
                                     placebo
```

Nummerical presets may or may not work still, but I would recommend using string names now.

And vvdec is surely the "original" command line decoder, best suited for testing purposes. If there is an issue, the reason may be most probably in the encoder (or in the multiplexer if you put the result into a container).

Is there an important reason to use so many threads and such a high frame parallelism explicitly, instead of allowing auto selection?

At least I can confirm that MPC-BE has issues playing a "placebo" encoded video with your (IMHO quite insane) values. Sample files:

[in_to_tree_720p50_uvg266.zip](https://www.ligh.de/test/in_to_tree_720p50_uvg266.zip) (using a standard sample Y4M from the Derf archive)
